### PR TITLE
Feature/new product description

### PR DESCRIPTION
**Pull request by mateuszkula**
I want to merge this change because...

It adds new product description with some small graphical fixes on product page.

Before:
Zrzut ekranu 2020-02-11 o 14 25 33

After:
Zrzut ekranu 2020-02-11 o 14 27 14

### CODE REVIEW

File: src/views/Product/Page.tsx [hunk 1]
```diff
@@ -14,6 +13,8 @@ import GalleryCarousel from "./GalleryCarousel";
 import OtherProducts from "./Other";
 import { ProductDetails_product } from "./types/ProductDetails";
 
+import { ProductDescription as NewProductDescription } from "../../@next/components/molecules";
```
Comment (orzechdev) [inline src/views/Product/Page.tsx]: May we just stay with ProductDescription? I assume it was just temporary change, am I right?

Comment (mateuszkula) [reply src/views/Product/Page.tsx]: There is a second component called ProductDescription(the old one) but I cant just throw it away and use new one instead as it does much more than the name suggests ;)

**Comment by Boquete** [comment]
suggestion: for variant attributes there should be option, or even as default, instead of drop-down list create boxes to select as in a-dam store. You can easily see there which variants are available.

**Comment by Boquete** [comment]
ad https://i.imgur.com/Ovzyr6p.png

**Pull request merged**